js/app/controllers/calcontroller.js

```diff
@@ -130,19 +130,37 @@ app.controller('CalController', ['$scope', '$rootScope', '$window', 'Calendar',
 			$scope.uiConfig.calendar.timezone = 'UTC';
 		});
 
-		CalendarService.getAll().then(function(calendars) {
-			$scope.calendars = calendars;
-			is.loading = false;
-			// TODO - scope.apply should not be necessary here
-			$scope.$apply();
-
-			angular.forEach($scope.calendars, function (calendar) {
-				$scope.eventSource[calendar.url] = calendar.fcEventSource;
-				if (calendar.enabled) {
+		var url = window.location.toString();
+		if (url.substr(url.length - 9) === 'calendar/' || url.substr(url.length - 8) === 'calendar' || url.substr(url.length - 10) === 'calendar/#') {
```

Comment:
mobile Safari supports it according to http://kangax.github.io/compat-table/es6/#test-String.prototype_methods_String.prototype.endsWith

I don't use Android, how common is it to use the stock Android browser?